subsys/net/ip/ipv6.c

```diff
@@ -1453,15 +1453,12 @@ static enum net_verdict handle_ns_input(struct net_pkt *pkt)
 
```

Comment:
Commit subject looks weird because of camel-case in ipv6, just do "net: ipv6: ...." here.

Reply:
yes, corrected.